components/editor/prosemirror-schema/modify-default-schema.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Modify the Default Schema
+page_title:  Modify the Default Schema
+description: Expore how to modify the default schema the Editor for Blazor uses.
+slug: editor-modify-default-schema
+tags: telerik,blazor,editor,prosemirror,schema
+published: True
+position: 3
+---
+
+# Modify the Default ProseMirror Schema
+
+This article describes how you can modify the default [ProseMirror schema that the Editor for Blazor uses]({%slug editor-prosemirror-schema-overview%}). Updating the existing schema is useful if you want to:
+
+* Extend the Editor capabilities and allow your users to add more kinds of HTML tags than the predefined ones.
+* Allow adding more attributes to the predefined HTML elements.
+* Restrict the users from adding some of the predefined HTML elements.
+
+@[template](/_contentTemplates/editor/general.md#prosemirror-schema-prerequisites)
+
+## Basics
```

Comment:
This section isn't getting rendered locally. See this screenshot:

![image](https://github.com/user-attachments/assets/f209a9a9-b87d-4fec-9e53-5d9571b02c8a)

Reply:
It is rendered correctly on my end.